faf-commons-data/src/main/java/com/faforever/commons/replay/shared/Utils.java

```diff
@@ -0,0 +1,97 @@
+package com.faforever.commons.replay.shared;
+
+import com.google.common.io.LittleEndianDataInputStream;
+import org.jetbrains.annotations.Contract;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Utils {
```

Comment:
I would probably rename to LoadUtils or ParseUtils

Reply:
Processed in https://github.com/FAForever/faf-java-commons/pull/143/commits/020955d09c208eef32ee7cec96f3e9314570a837